src/App.js

```diff
@@ -1,13 +1,17 @@
-import './App.css';
+import React from 'react'
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Home from './components/Home';
+import Categories from './components/Categories';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <BrowserRouter>
+    <Header />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/categories" element={<Categories />} />
+    </Routes>
+  </BrowserRouter>
+);
 
```

Comment:
- [ ] - [ ]  Your app runs with errors. I'm not able to see the home page.  After running the `npm install` then `npm start`.  Actually, I'm getting this error message 👇🏻 Please ensure that your app works as expected and runs without errors. 😉 👍🏻 

- Browser: 
![0 2](https://user-images.githubusercontent.com/79058364/181259442-e919f4d8-4067-4375-9dc0-1c8cf0b8cf10.png)

Reply:
Thank you for the review and thorough feedback. Aside from the optional change request and the README everything appears to work perfectly well from my end. I really can't understand why you are getting the error.